examples/with-xata/README.md

```diff
@@ -0,0 +1,70 @@
+<h1> Next.js + Xata Minimal Template <img alt="Xatafly, the logo from Xata" src="/public/flap.gif" width="50" /></h1>
```

Comment:
Can we use the same README format as the other examples, please? 🙏

Reply:
I just pushed some changes to the Readme, I think now it's more aligned with the rest of the examples